src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3071,9 +3216,19 @@ void JasmineGraphInstanceService::createPartitionFiles(std::string graphID, std:
                         "_centralstore_" + partitionID;
         outputFilePath = utils.getJasmineGraphProperty("org.jasminegraph.server.instance.trainedmodelfolder") + "/" +
                          graphID + "_centralstore_" + partitionID;
+
+        std::ifstream infile(inputFilePath);
+        int a, b;
+        while (infile >> a >> b)
+        {
+            instance_logger.log(to_string(a) + " " + to_string(b), "info");
```

Comment:
Why do we log the entire edge list here?

Reply:
Fixed

Reply:
Ok